artifactory_test.go

```diff
@@ -120,8 +126,10 @@ func testSingleTechDependencyResolution(t *testing.T, testProjectPartialPath []s
 		Url:            *securityTests.JfrogUrl,
 		ArtifactoryUrl: *securityTests.JfrogUrl + securityTests.ArtifactoryEndpoint,
 		XrayUrl:        *securityTests.JfrogUrl + securityTests.XrayEndpoint,
-		AccessToken:    *securityTests.JfrogAccessToken,
-		ServerId:       securityTests.ServerId,
+		//AccessToken:    *securityTests.JfrogAccessToken,
```

Comment:
```suggestion
		AccessToken:    *securityTests.JfrogAccessToken,
```
Tests are using access token, we are not providing a user and password. as far as I know maven should work with it

Reply:
sure